README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://github.com/dmlc/dlpack/actions/workflows/main.yaml/badge.svg?branch=main)](https://github.com/dmlc/dlpack/actions/workflows/main.yaml)
 
+Documentation: [https://dmlc.github.io/dlpack/latest](https://dmlc.github.io/dlpack/latest)
```

Comment:
worth adding that link in the right side of the gh repo under project description @tqchen

Reply:
I agree, that would be helpful!

Reply:
updated!

Reply:
Thanks @tqchen!